### ISSUE TITLE
Command 'list' should respect 'exclude', 'include', and 'prepend'

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/port4me#14]

**Comment by HenrikBengtsson** [comment]
```sh
$ port4me --list=5 --prepend=1024-1050
1024
1025
1026
1027
1028

$ port4me --list=5 --include=1024-1050
1047
1026
1048
1050
1046

$ port4me --list=5 --prepend=1024-1050 --exclude=1024-1030
1031
1032
1033
1034
1035
```

**Comment by HenrikBengtsson** [comment]
```r
$ Rscript -e "port4me::port4me(list=5, prepend=1024:1050)"
[1] 1024 1025 1026 1027 1028
```

However,

```r
$ Rscript -e "port4me::port4me(list=5, include=1024:1050)"
Error in port4me::port4me(list = 5, include = 1024:1050) : 
  Failed to find a free TCP port after 5 attempts

$ Rscript -e "port4me::port4me(list=5, prepend=1024:1050, exclude=1024:1030)"
Error in port4me::port4me(list = 5, prepend = 1024:1050, exclude = 1024:1030) : 
  Failed to find a free TCP port after 5 attempts
```

**Comment by HenrikBengtsson** [comment]
Fixed in port4me (>= 0.3.0-9008);

```r
$ Rscript -e "port4me::port4me(list=5, prepend=1024:1050)"
[1] 1024 1025 1026 1027 1028

$ Rscript -e "port4me::port4me(list=5, include=1024:1050)"
[1] 1047 1026 1048 1050 1046

$ Rscript -e "port4me::port4me(list=5, prepend=1024:1050, exclude=1024:1030)"
[1] 1031 1032 1033 1034 1035
```
